git-extra/vimrc

```diff
@@ -30,6 +30,12 @@ if &term =~ 'xterm'
   let &t_SI.="\e[5 q"           " A thin blinking bar for insert/replace mode
   let &t_EI.="\e[1 q"           " block cursor after leaving insert/replace mode
   let &t_te.="\e[0 q"           " Blinking block cursor when leaving termcap mode
+  if &t_Co == 8
+    set t_Co = 256              " Use at lease 256 colors
+  endif
+  if 0
+    set termguicolors           " Allow trucolors on mintty 
+  endif
```

Comment:
What is the effect of this? The commit message stops short of explaining that.

Also, the commit message starts with an unnecessarily controversial "better". Let's avoid that. Let's just call it "Use more colors".

Reply:
will do